src/Plugins/LevelDBStore/IO/Data/LevelDB/DB.cs

```diff
@@ -37,12 +41,39 @@ public void Dispose()
             }
         }
 
+        /// <summary>
+        /// Remove the database entry (if any) for "key".
+        /// It is not an error if "key" did not exist in the database.
```

Comment:
Relation to `Delete(WriteOptions options, byte[] key)` should be mentioned.

Reply:
now new methods being removed, this will purly be a comments/style pr.

Reply:
@Jim8y, it's all about consistency of what we do here and what we do there. Is this method bad on its own? No, it's a nice method. But if your PR says "comments and style" I expect to see comments and style fixes, while this is a functional change. It can be a separate commit to me (but looks like proper commit management is a lost art these days), it can be a separate PR, but it just doesn't belong to "comments and style".

Try using `git log` or `git blame` in this repository for any purpose --- you'll quickly discover what I'm talking about.

Reply:
I apologize for my previous rude comment, it exists for like 10 seconds then i deleted it, I felt it unnecessary to be so strict on few line change, but now i totally agree with you. Pr being fully updated according to your comment and suggestion, sorry agian for my ignorance.